src/ray/raylet/scheduling/cluster_resource_scheduler.cc

```diff
@@ -115,7 +107,7 @@ scheduling::NodeID ClusterResourceScheduler::GetBestSchedulableNode(
     bool *is_infeasible) {
   // The zero cpu actor is a special case that must be handled the same way by all
   // scheduling policies.
-  if (actor_creation && resource_request.IsEmpty()) {
```

Comment:
cc @jjyao for this change.

Reply:
I feel we can decouple this change from this PR, seems unrelated?